content/en/docs/tasks/manage-daemon/create-daemon-set.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Building a Basic DaemonSet  
+content_type: task  
+weight: 30  
+---
+<!-- overview -->
+
+This page demonstrates how to build a basic DaemonSet that runs a Pod on every node in a Kubernetes cluster. It covers a simple use case of mounting a file from the host, logging its contents using an init container, and utilizing a pause container.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+## Building a DaemonSet
+
+In this task, you'll create a basic DaemonSet that mounts the `/etc/machine-id` file from each node, logs its contents using an init container, and runs a pause container.
+
+### Step 1: Create the DaemonSet Manifest
+
+Let's create a DaemonSet manifest that ensures one Pod is scheduled on every node. The Pod will use an init container to read and log the contents of `/etc/machine-id` from the host, while the main container will be a `pause` container, which keeps the Pod running.
+
+```yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: example-daemonset
+spec:
+  selector:
+    matchLabels:
+      app: example
+  template:
+    metadata:
+      labels:
+        app: example
+    spec:
+      containers:
+      - name: pause
+        image: gcr.io/google_containers/pause:3.1
+        # This is the main container, which doesn't do much but keeps the Pod alive.
+      initContainers:
+      - name: log-machine-id
+        image: busybox
+        command: ['sh', '-c', 'cat /etc/machine-id > /var/log/machine-id.log']
+        volumeMounts:
+        - name: machine-id
+          mountPath: /etc/machine-id
+          readOnly: true
+        - name: log-dir
+          mountPath: /var/log
+      volumes:
+      - name: machine-id
+        hostPath:
+          path: /etc/machine-id
+          type: File
+      - name: log-dir
+        hostPath:
+          path: /var/log
+```
+
+### Step 2: Apply the DaemonSet
+
+Apply the DaemonSet to your Kubernetes cluster by saving the manifest to a file (For example, `daemonset.yaml`) and using the following command:
```

Comment:
Instead of mentioning raw yaml script, its good if you can place the yaml file in code samples and reference here.

Reply:
@T-Lakshmi I wanted to understand so here we should explain the particular code blocks instead of giving the whole yaml ?

Reply:
In the task doc I see usually the raw yaml is mentioned for example https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/ 
I tried to follow the same structure, can you please elaborate the new change if possible that would be really helpful ?

Reply:
Have a look at https://kubernetes.io/search/?q=code_sample; that should help you find advice on what to do.

You can - and should - look at how other task pages tackle this.